### PR TITLE
Launch new categories page

### PR DESCRIPTION
**Pull request by anandaroop**
This officially launches the new /categories page, aka the public-facing homepage for The Art Genome Project.

categories app

### CODE REVIEW

**Comment by starsirius** [comment]
Woohoo! Excited!! 🎉

File: desktop/apps/categories/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Categories app
```
Comment (craigspaeth) [inline desktop/apps/categories/README.md]: 👌  nice readme!

File: desktop/apps/categories/routes.coffee [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
-Items = require '../../collections/items'
-Genes = require '../../collections/genes'
-OrderedSets = require '../../collections/ordered_sets'
-
-@index = (req, res) ->
-  featuredGenes = new Items [], id: '51ba3bd10abd8521b3000049'
-  geneCategories = new OrderedSets key: 'browse:gene-categories'
-  genes = new Genes
-
-  Q.all([
```
Comment (craigspaeth) [inline desktop/apps/categories/routes.coffee]: 🔥 bye Q, hello async/await!

**Comment by anandaroop** [comment]
Awesome, thanks Craig!

**Pull request merged**